src/test/polymer/polymer-element_test.ts

```diff
@@ -17,22 +17,33 @@ import {assert} from 'chai';
 import * as path from 'path';
 
 import {Analyzer} from '../../analyzer';
+import {HtmlScriptScanner} from '../../html/html-script-scanner';
 import {ClassScanner} from '../../javascript/class-scanner';
-import {Document} from '../../model/model';
+import {Document, Severity, Warning} from '../../model/model';
 import {PolymerElement} from '../../polymer/polymer-element';
+import {PolymerElementScanner} from '../../polymer/polymer-element-scanner';
 import {FSUrlLoader} from '../../url-loader/fs-url-loader';
 
 suite('PolymerElement', () => {
   const testFilesDir = path.resolve(__dirname, '../static/polymer2/');
   const urlLoader = new FSUrlLoader(testFilesDir);
   const analyzer = new Analyzer({
     urlLoader: urlLoader,
-    scanners: new Map([[
-      'js',
+    scanners: new Map<string, any[]>([
```

Comment:
IMO building an analyzer with a subset of scanners is an antipattern, because what we care about is how the analyzer works with all the scanners.

Cards on the table, I tend to see integration tests as the ideal, and unit tests as compromises for speed or clarity. Reasonable people may disagree :)